<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1667565937777' stop='1667565947029'>
    <name>btlp workflow</name>
    <title>btlp workflow</title>
    <test-cases>
        <test-case start='1667565937777' status='broken' stop='1667565947028'>
            <name>log in into WMS</name>
            <title>log in into WMS</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-107.0.5304.87'/>
            </parameters>
            <steps>
                <step start='1667565937777' status='broken' stop='1667565947028'>
                    <name>Maximize the window</name>
                    <title>Maximize the window</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>socket hang up</message>
                <stack-trace>RequestError: socket hang up
                    at ClientRequest.&lt;anonymous> (/Users/shrinathmotappamirashi/Documents/seenu/qa-automation-oms/oms/oos/node_modules/got/dist/source/core/index.js:970:111)
                    at Object.onceWrapper (node:events:642:26)
                    at ClientRequest.emit (node:events:539:35)
                    at ClientRequest.emit (node:domain:475:12)
                    at ClientRequest.origin.emit (/Users/shrinathmotappamirashi/Documents/seenu/qa-automation-oms/oms/oos/node_modules/@szmarczak/http-timer/dist/source/index.js:43:20)
                    at Socket.socketOnEnd (node:_http_client:478:9)
                    at Socket.emit (node:events:539:35)
                    at Socket.emit (node:domain:475:12)
                    at endReadableNT (node:internal/streams/readable:1345:12)
                    at connResetException (node:internal/errors:692:14)
                    at Socket.socketOnEnd (node:_http_client:478:23)
                    at Socket.emit (node:events:539:35)
                    at Socket.emit (node:domain:475:12)
                    at endReadableNT (node:internal/streams/readable:1345:12)
                    at processTicksAndRejections (node:internal/process/task_queues:83:21)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>